### PR TITLE
Create additional presets

### LINKED ISSUE #204: Add more Presets for Shooting
tuning time - potentially more presets
Shooter Subsystem

### PR DESCRIPTION
**Pull request by AlenaNguyen**

### CODE REVIEW

File: src/main/java/frc/robot/commands/ShootingPreset.java [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+  public void initialize() {
+  }
+
+  // Called every time the scheduler runs while the command is scheduled.
+  @Override
+  public void execute() {
+    subShooter.setDesiredVelocities(desiredLeftVelocity, desiredRightVelocity);
+    subTurret.setTurretAngle(desiredTurretAngle);
+    subPitch.setPitchAngle(desiredPitchAngle);
+    subShooter.setIgnoreFlywheelSpeed(ignoreFlywheelSpeed);
```
Comment (TaylerUva) [inline src/main/java/frc/robot/commands/ShootingPreset.java]: These should all be done in initialize and we need to run shooter.getuptospeed

Comment (TaylerUva) [inline src/main/java/frc/robot/commands/ShootingPreset.java]: Don't forget to set it back to normal in end

**Review by TaylerUva — CHANGES_REQUESTED** [review]

**Comment by alicekuznetsov** [comment]
image

**Comment by alicekuznetsov** [comment]
as much as we need to rename the gulf of Mexico, you all must understand that i have 2 days to learn these controls and i will not be physically capable of forgetting "gulf of Mexico" unless the new name is even funnier. this is very difficult task but we must over come it

**Comment by AlenaNguyen** [comment]
good news: the presets are presetting and Alice did not explode the robot by changing the button mapping 🎉

**Review by TaylerUva — APPROVED** [review]

**Pull request merged**